compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -1150,49 +1154,6 @@ declare_lint! {
     "detects attempts to mutate a `const` item",
 }
 
-declare_lint! {
```

Comment:
Is it okay to entirely remove the lints after they make their way to stable? I mildly recall us having a mechanism (much like for features) to "remove" a lint without actually removing it and/or making things like `#[deny(warnings, safe_packed_borrows)]` fail.

Reply:
[`register_renamed`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_lint/context/struct.LintStore.html#method.register_renamed) and [`register_removed`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_lint/context/struct.LintStore.html#method.register_removed)

Reply:
Oh right, good point.